sdjournal/read.go

```diff
@@ -38,7 +38,10 @@ type JournalReaderConfig struct {
 	// the array is empty, entries will not be filtered.
 	Matches []Match
 
-	MaxEntryLength uint64 // the max allowable length for an entry (default is 64K)
+	// explicitly set MaxEntryLenght to 0, which means unlimited
+	DisableMaxEntryLength bool
+	// the max allowable length for an entry (default is 64K)
+	MaxEntryLength uint64
```

Comment:
disabling this was tricky because unless this becomes a pointer there's no way of saying if the used want to disable it because it will default to 0 when the struct is initialized. I thought about creating a `NewJournalReaderConfig` but was kind of ugly.